js/directives/lucca/user-picker.js

```diff
@@ -544,14 +539,13 @@
 				}
 			});
 
-			$http.get(query).then(
-				function(response) {
-					var homonyms = response.data.data.items;
-					deferred.resolve(homonyms);
-				}, function(message) {
-					deferred.reject(message);
-				}
-			);
+			$http.get(query)
+			.then(function(response) {
+				var homonyms = response.data.data.items;
+				deferred.resolve(homonyms);
+			}, function(message) {
```

Comment:
does it really rejects with just a message ? the [doc](https://docs.angularjs.org/api/ng/service/$http) seems to indicate that the promise rejects with an object response with a member `status` and i gues a member `data`